src/wallet/wallet.cpp

```diff
@@ -924,6 +924,42 @@ void CWallet::LoadToWallet(CWalletTx& wtxIn)
     }
 }
 
+void CWallet::RemoveOldTransactions(const CScript& script)
+{
+    // AssertLockHeld(cs_wallet);
```

Comment:
Why is this entire function commented out?

Reply:
woops, pushed wrong commit. Should be good now